glue/viewers/image/qt/slice_widget.py

```diff
@@ -89,11 +89,10 @@ def sync_sliders_from_state(self, *args):
                 # check the type of data.coords here since we want to treat
                 # subclasses differently.
                 if getattr(self.data, 'coords') is not None and type(self.data.coords) != LegacyCoordinates:
-                    world_axis_index = self.data.ndim - 1 - i
                     world = world_axis(self.data.coords, self.data,
-                                       pixel_axis=world_axis_index,
-                                       world_axis=world_axis_index)
-                    world_unit = self.data.coords.world_axis_units[world_axis_index]
+                                       pixel_axis=self.data.ndim - 1 - i,
+                                       world_axis=self.data.ndim - 1 - i)
+                    world_unit = self.data.coords.world_axis_units[self.data.ndim - 1 - i]
```

Comment:
Can you revert these changes since the original code is a bit more readable?

Reply:
Sure